libs/langchain/langchain/chat_models/databricks.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from typing import List, Optional
+
+from langchain_core.pydantic_v1 import BaseModel, Extra
+
+from langchain.chat_models.mlflow import ChatMlflow
+
+logger = logging.getLogger(__name__)
+
+
+# Ignoring type because below is valid pydantic code
+# Unexpected keyword argument "extra" for "__init_subclass__" of "object"  [call-arg]
+class ChatParams(BaseModel, extra=Extra.allow):  # type: ignore[call-arg]
+    """Parameters for the `MLflow` LLM."""
+
+    temperature: float = 0.0
+    n: int = 1
+    stop: Optional[List[str]] = None
+    max_tokens: Optional[int] = None
```

Comment:
I feel this can be dropped as we will use the `ChatParams` in `chat_models/mlflow.py`, right?

Reply:
Yeah we can drop it.